dataprofiler/labelers/data_processing.py

```diff
@@ -614,7 +615,11 @@ def process(self, data, labels=None, label_mapping=None, batch_size=32):
 
         # Import tensorflow
         import tensorflow as tf
-        
+
+        if data.ndim > 1:
+            data = data.reshape(-1)
+        data = data.astype(str)
```

Comment:
Could this cause conflicts of some kind?

Reply:
do we really need it, as the function below this does have the processing for this already, i.e., `sample_buffer = str(sample_buffer)`